game.rb

```diff
@@ -24,21 +23,28 @@ def player_one_pick_word
   end
 
   def player_two_guess_word word
-    counter = 0
+    guess_counter = 0
     end_of_game = false
+    @io_handler.print(@game_view.body_array)
```

Comment:
The way you have this makes it difficult to test this is happening without messing with STDOUT. it would be nice if this was it's own method so you could at least make sure it was being called.